libmscore/figuredbass.cpp

```diff
@@ -17,10 +17,13 @@
 #include "system.h"
 #include "segment.h"
 #include "chord.h"
+#include "staff.h"
 #include "rest.h"
 #include "score.h"
 #include "sym.h"
 #include "xml.h"
+#include "utils.h"
+#include "mscore/scoreview.h"
```

Comment:
you can't do that as far as I know. You can share a link like you did. Or you can copy paste the code and use Markdown syntax for code.
What I show is not for quoting but to comment on a particular line of code.